service/src/main/java/tennisclub/service/TournamentService.java

```diff
@@ -0,0 +1,19 @@
+package tennisclub.service;
+
+import tennisclub.entity.Tournament;
+import tennisclub.entity.ranking.Ranking;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
```

Comment:
In the reference eshop project they used the @Service annotation also for the interface, I guess we should add it here as well.

Reply:
Thanks for pointing this out! 👍